tests/fortex/health/readers/xray_image_reader_test.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for XrayImageReader
+"""
+import sys
+
+sys.path.insert(0, "../../../../")
```

Comment:
I am not sure if I correctly understand necessity of this here, is there a module that is inaccessible without this in this file?

Reply:
yes without this, I was unable to import the module.

Reply:
Oh alright, which  module import fails without this?

Reply:
XrayImageReaderDir itself is unable to import from the examples folder.

Reply:
I am a little apprehensive of this line, as future processors/reader writers would refer existing processors/readers as a template and this doesn't seem like the best way to import data samples from `/example`s folder. Trying out a change, will merge this PR once that works.